packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -867,6 +888,12 @@ public String stopVideoRecording() {
     }
     String path = captureFile.getAbsolutePath();
     captureFile = null;
+
+    // if there was an RuntimeException during stopping the video recording, return null to let the
```

Comment:
This comment doesn't say anything that the code it is commenting doesn't trivially indicate (other than claiming that it's because of a RuntimeException, which could easily become incorrect over time), so it can just be removed.